thoth/storages/graph/postgres.py

```diff
@@ -3475,6 +3475,15 @@ def update_kebechet_github_installations_on_is_active(self, slug: str) -> bool:
                 return True
             return False
 
+    def get_kebechet_github_installations_active(self) -> List[str]:
```

Comment:
please read naming convention for queries: https://github.com/thoth-station/storages#query-naming-conventions-in-thoth

Reply:
thx, didn't notice that!